vispy/visuals/histogram.py

```diff
@@ -22,37 +23,63 @@ class HistogramVisual(MeshVisual):
         Color of the histogram.
     orientation : {'h', 'v'}
         Orientation of the histogram.
+    calc_hist : callable
```

Comment:
`callable` or `Callable`? If you or napari or some other project are OK with only type annotations I'm OK slowly migrating to that.